test/evergreen.yml

```diff
@@ -173,13 +173,16 @@ functions:
       script: |
         set -o errexit
         set -o verbose
+        dir=$(pwd)
+        cd wiredtiger
         mongo_repo=https://github.com/mongodb/mongo
-        branch=${branch_name}
+        branch=$(git branch --contains | grep -e "mongodb" -e "develop")
         if [[ $branch =~ "mongodb-" ]]; then
           mongo_branch=v$(echo $branch | cut -d'-' -f 2)
         else
           mongo_branch=master
         fi
+        cd $dir
```

Comment:
This can be simplified with either `pushd`/`popd` or `cd -` (no need for `$dir`).

Reply:
Thanks! Changed to use pushd and popd